src/styles.css

```diff
@@ -229,7 +229,38 @@ button {
   color: inherit;
   text-decoration: none;
 }
+
 .pmndrs-menu a:hover {
   text-decoration: underline;
   color: inherit;
 }
+
+.nosound {
+  position: absolute;
+  top: 20px;
+  right: 20px;
+  height: 24px;
+  width: 24px;
+  background: url(./textures/no_sound.png) no-repeat;
+  background-size: 24px 24px;
+  opacity: 1;
+  animation: hideSoundIcon 3s forwards;
+}
+
+@keyframes hideSoundIcon {
```

Comment:
useSpring might be perfect for this!

Reply:
css much faster as to instance an extra hook for a small ani

Reply:
i agree, usespring has per frame overhead, we dont have that much movement in the dom so i think css could be good enough.